src/Symfony/Component/Translation/TranslatorInterface.php

```diff
@@ -45,7 +45,7 @@ public function trans($id, array $parameters = array(), $domain = null, $locale
      *
      * @throws \InvalidArgumentException If the locale contains invalid characters
      */
-    public function transChoice($id, $number, array $parameters = array(), $domain = null, $locale = null);
+    public function transChoice($id, $number = 1, array $parameters = array(), $domain = null, $locale = null);
```

Comment:
Thanks. I was also toying with the idea of just modifying the call to `transChoice` in the twig-bridge `transchoice` filter, thereby satisfying the aims, but not breaking BC.